### ISSUE TITLE
Upload Method

### ISSUE THREAD

**Issue by davidchalifoux** [putdotio/putio.js#7]
Could you add an `Upload` method to the File or Files class? This would really be useful on my CLI client.
I would understand if you consider it to be outside of the scope of the library though.

**Comment by altayaydemir** [comment]
Nice idea, I may expose the `uploader` utility.

**Comment by boulosda** [comment]
Until it is added to the SDK, if you need to upload I have written this snippet to help:

```
const FormData = require('form-data');

const PutioAPI = require('@putdotio/api-client').default;

const putioAPI = new PutioAPI({ clientID: '' });
putioAPI.setToken('');
putioAPI.Files.Upload = function({
  file,
  filename = '',
  parent_id = ''
}) {
  const form = new FormData();
  form.append('file', file);
  form.append('filename', filename);
  form.append('parent_id', parent_id);
  return this.client.post('/files/upload', {
    data: form,
    headers: form.getHeaders()
  });
}

// usage
putioAPI.Files.Upload({
  file: require('fs').createReadStream('/path/to/torrent/file.torrent')
})
.then(resp => {})
.catch(err => {});
```

**Comment by altayaydemir** [comment]
@boulosda that looks good :) would you like to create a PR?

**Comment by boulosda** [comment]
#15 PR created :)

**Comment by altayaydemir** [comment]
released in [v8.12.0](https://github.com/putdotio/putio.js/releases/tag/v8.12.0)
